beamer/config.py

```diff
@@ -20,9 +20,7 @@ class ConfigError(Exception):
 class Config:
     account: LocalAccount
     deployment_info: DeploymentInfo
-    l1_rpc_url: URL
-    l2a_rpc_url: URL
-    l2b_rpc_url: URL
+    chains: dict[str, URL]
```

Comment:
I would rename this attribute to something like `rpc_urls` -- `chains` is misleading.